packages/app-earn-ui/src/features/google-tag-manager/GoogleTagManager.tsx

```
@@ -0,0 +1,36 @@
+import Script from 'next/script'
+
+const GTM_ID = 'GTM-PWFF39MK'
+
+export const GoogleTagManager: FC = () => {
+  return (
+    <>
+      <Script
+        id="gtm-script"
+        strategy="afterInteractive"
+        type="text/javascript"
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)